app/controllers/v0/dependents_applications_controller.rb

```diff
@@ -21,7 +21,12 @@ def create
     end
 
     def show
-      render(json: DependentsApplication.find(params[:id]))
+      dependent_service = BGS::DependentService.new
+      response = dependent_service.get_dependents(current_user)
+      render json: response, each_serializer: DependentsSerializer
+    rescue => e
```

Comment:
Not sure if this should really rescue all exceptions, or just the BGS Share exception that is raised.

Reply:
I tend to agree

Reply:
@johnpaulashenfelter  since this is raised to sentry still, I am ok with the catch all, because it returns a properly formed response vs a 500 and the client having to handle that. What do you think? I am not against being specific either and having better errors.